tests/apple-exporter/components/test-class-body.php

```diff
@@ -525,4 +526,76 @@ public function test_without_dropcap() {
 		$this->assertEquals( '<p>Paragraph 2.</p>', $json['components'][3]['text'] );
 		$this->assertEquals( 'default-body', $json['components'][3]['textStyle'] );
 	}
+
+	/**
+	 * A data provider that arguments for `test_dropcap_determination` tests.
+	 *
+	 * @return array An array of arrays representing function arguments.
+	 */
+	public function data_dropcap_determination() {
```

Comment:
Nice to have the data provider grouped with the test itself for readability but I can move this to the top of the test class to keep with conventions if needed.

Reply:
I'd like to keep the function names in alphabetical order to make it easier to find things within the file. Sometimes we use the same data provider across multiple tests, as well.

Reply:
Sounds good, didn't realize the tests were alphabetized.  I'll reorder these.